src/main/java/mate/academy/spring/MainApp.java

```diff
@@ -0,0 +1,114 @@
+package mate.academy.spring;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import mate.academy.spring.config.AppConfig;
+import mate.academy.spring.entity.Author;
+import mate.academy.spring.entity.Book;
+import mate.academy.spring.entity.User;
+import mate.academy.spring.service.BookService;
+import mate.academy.spring.service.UserService;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class MainApp {
+    public static void main(String[] args) throws SQLException {
+        AnnotationConfigApplicationContext context =
+                new AnnotationConfigApplicationContext(AppConfig.class);
+
+        UserService userService = context.getBean(UserService.class);
+        BookService bookService = context.getBean(BookService.class);
+   //     AuthorService authorService = context.getBean(AuthorService.class);
+
+        User user_1 = new User("Sunil", "Bora", "suni.bora@example.com");
```

Comment:
@Sasha192 mvn clean package ?
Please use more useful variable names: 
```suggestion
        User userSunilBora = new User("Sunil", "Bora", "suni.bora@example.com");
```